net/upnp/src/www/services_upnp.php

```diff
@@ -282,7 +288,9 @@ function miniupnpd_validate_port($port)
                     <tr>
                       <td><a id="help_for_logpackets" href="#" class="showhelp"><i class="fa fa-info-circle"></i></a> <?=gettext("Log NAT-PMP");?></td>
                       <td>
-                       <input name="logpackets" type="checkbox" value="yes" <?=!empty($pconfig['logpackets']) ? "checked=\"checked\"" : ""; ?> />
+                        <label for="logpackets">
+                          <input name="logpackets" type="checkbox" name="logpackets" value="yes" <?=!empty($pconfig['logpackets']) ? "checked=\"checked\"" : ""; ?> />
```

Comment:
missing "id" attribute here, the "for" attribute linking is mapped on "id" attribute.
Oh in fact there is twice names

Reply:
Thanks.  Fixed.